tests/kernel/kernel_constructors.cpp

```diff
@@ -54,16 +53,18 @@ TEST_CASE("Test copy constructor", "[kernel]") {
     INFO(
         "kernel was not constructed correctly. (contains different "
         "OpenCL kernel object)");
-    CHECK((ctsSelector.is_host() || (iopKernelA == iopKernelB)));
+    CHECK((sycl::device(cts_selector)
+                   .get_info<sycl::info::device::device_type>() ==
+               sycl::info::device_type::host ||
+           (iopKernelA == iopKernelB)));
```

Comment:
This test has to be updated since the concept of `host` does not exist anymore in SYCL 2020.

Reply:
Removed safeguards for host in kernel tests.